code/ship/ship.cpp

```diff
@@ -3729,6 +3742,18 @@ static void parse_ship_values(ship_info* sip, const bool is_template, const bool
 			stuff_float(&sip->afterburner_recover_rate);
 		}
 
+		if (optional_string("+Aburn Min Fuel to Start:")) {
+			stuff_float(&sip->afterburner_min_fuel_to_start);
+		}
+
+		if (optional_string("+Aburn Min Fuel to Consume:")) {
```

Comment:
"+Afterburner Minimum Fuel Consumption:" ?

Reply:
All better names, thanks it was kind of awkward, although I think ill keep it to "Aburn" to match the other afterburner options.

Not sure what can be done about this one to avoid implying that it will initially take this amount or something, other than just the documentation.

Reply:
"Min fuel to be burned" maybe? "To burn" would be punchier and still imply the right thing, maybe. There's more room for ambiguity but "to start" existing next to it maybe would avoid that?